### PR TITLE
[helm/charts] Add storages

### PR DESCRIPTION
**Pull request by faizanahmad055**
Signed-off-by: Faizan Ahmad
username: faizanahmad055
email: faizan.ahmad55@outlook.com

#### What this PR does / why we need it:
Storages chart is used in conjunction with other charts. It can come in handy when a public chart does not provide support for the additional pv and pvc. In that scenario, this chart can be used to deploy additional pv and pvc.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/storages/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# storages
+
+This chart is used to apply pv and pvc. You can specify any number of pv and pvc. This chart can come in handy when a public chart does not offer additional pv and pvc. You have to use the following format to specify pv and pvc:
+
+```bash
```
Comment (philoserf) [inline stable/storages/README.md]: YAML not shell

File: stable/storages/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# storages
+
+This chart is used to apply pv and pvc. You can specify any number of pv and pvc. This chart can come in handy when a public chart does not offer additional pv and pvc. You have to use the following format to specify pv and pvc:
```
Comment (philoserf) [inline stable/storages/README.md:3]: README need more context, perhaps a use case of combining this with the “public chart”.

File: stable/storages/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+storages chart is used with conjunction of other charts. It can come in handy when a public chart does not provide support for additional pv or pvc. In that scenario, this chart can be used to deploy additional pv and pvc.
```
Comment (philoserf) [inline stable/storages/templates/NOTES.txt]: NOTES.txt is intended for post installation messages. This is a restrment of purpose.

**Review by philoserf — CHANGES_REQUESTED** [review]

File: stable/storages/README.md (hunk 1 above)
Comment (faizanahmad055) [reply stable/storages/README.md]: Updated

File: stable/storages/README.md (hunk 2 above)
Comment (faizanahmad055) [reply stable/storages/README.md:3]: Updated with an example

File: stable/storages/templates/NOTES.txt (hunk 3 above)
Comment (faizanahmad055) [reply stable/storages/templates/NOTES.txt]: Updated

**Comment by faizanahmad055** [comment]
Hi @philoserf,
I've implemented the review comments. Please take a look.

**Comment by faizanahmad055** [comment]
Hi @philoserf,
Ping

**Comment by philoserf** [comment]
I can’t move the pull request along @faizanahmad055. Like you, I am a member of the community. I’m not an approver.

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Ping

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Ping

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Can you guys take a look? All requested changes have been implemented. If there are any more; please do let me know.

**Comment by faizanahmad055** [comment]
Hi @foxish and @mgoodness
Ping

**Comment by paulczar** [comment]
If this chart already exists and has a repo (which https://github.com/stakater-charts/storage seems to suggest is the case ).   You can add your own repo to helm hub by following these docs - https://github.com/helm/hub/blob/master/Repositories.md

**Comment by faizanahmad055** [comment]
Hi @paulczar,
This is a good point. But we are moving away from our repository and we want to remove the chart dependency from our repo. So that's why we opted to go for helm/charts repo.

**Comment by faizanahmad055** [comment]
Hi @davidkarlsen and @paulczar
Can you guys please take a look at this.

**Comment by davidkarlsen** [comment]
See https://github.com/helm/charts/pull/10248 - same argument

**Comment by faizanahmad055** [comment]
I think you are right about that. We are also preparing our public repository so it can be searchable at [hub.helm.sh](https://hub.helm.sh/). So we can host these chart at that repository and if anyone needs them, they can get it from there.

**Comment by paulczar** [comment]
I believe we're in agreement that you'll look to host these in your own repo and share that with the public helm hub.  I'll close this out for that.  If you want to reopen and discuss feel free to reach out on Kubernetes Slack.

**Pull request closed without merging**